modules/adnuntiusRtdProvider.js

```diff
@@ -0,0 +1,97 @@
+import { getGlobal } from '../src/prebidGlobal.js';
+import { submodule } from '../src/hook.js'
+import { logError, logInfo } from '../src/utils.js'
+import { ajax } from '../src/ajax.js';
+
+const GVLID = 855;
+
+function init(config, userConsent) {
+  if (!config.params) return false
+  if (!config.params.providers) return false
+  logInfo(userConsent)
+  return true;
+}
+
+// Make sure that ajax has a function as callback
+function prepProvider(provider) {
+  // Map parameter to something that adnuntius endpoint understands.
+  const mappedParameters = {
+    siteId: 's',
+    userId: 'browserId',
+    browserId: 'browserId',
+    folderId: 'folderId'
+  }
+
+  const tzo = new Date().getTimezoneOffset();
+  const URL = ['https://data.adnuntius.com/usr?tzo=' + tzo]
+  Object.keys(provider).forEach(key => {
+    URL.push(`${mappedParameters[key]}=${provider[key]}`)
+  })
+
+  return new Promise((resolve, reject) => {
+    ajax(URL.join('&'), {
+      success: function (res) {
+        const response = JSON.parse(res)
+        resolve(response)
+      },
+      error: function (err) { reject(err) }
+    });
+  });
+}
+
+function setGlobalConfig(config, segments) {
```

Comment:
This function should be merging ortb2 data not overwriting. There can very well be other useful segment data being set that this function would inadvertently remove. There should be usable mergeConfig/mergeBidderConfig functionality within src/config. You may need to merge in some commits from master though.

Reply:
Fixed in new pull request. :)